docs/extensibility.md

```diff
@@ -1,23 +1,22 @@
 
-## The mythical Loch Ness resource
-To demonstrate how new protocols can be added, we will create a protocol to discover Nessie, a mythical Loch Ness monster that lives at a specific url.
+To add a new protocol implementation, several things are needed:
 
-For reference, we have created a [nessie branch](https://github.com/deislabs/akri/tree/nessie) with the implementation defined below.  For convenience, you can [compare the nessie branch with main here](https://github.com/deislabs/akri/compare/nessie).
+1. Add a new DiscoveryHandler implementation in Akri Agent
+1. Update Configuration CRD to include the new DiscoveryHandler implementation
+1. Build versions of Akri agent and controller that understand the new DiscoveryHandler
+1. Create a (protocol) Broker for the new capability
 
-### Container Registry Setup
-Any docker-compatible container registry should work (dockerhub, Github Container Registry, Azure Container Registry, etc).
+This document is intended to demonstrate how a new protocol can be implemented.For reference, we have created a [http-extensibility branch](https://github.com/deislabs/akri/tree/http-extensibility) with the implementation defined below.  For convenience, you can [compare the http-extensibility branch with main here](https://github.com/deislabs/akri/compare/http-extensibility).
 
-For this sample, we are using the [GitHub container registry](https://github.blog/2020-09-01-introducing-github-container-registry/). You can follow the [getting started guide here to enable it for yourself](https://docs.github.com/en/free-pro-team@latest/packages/getting-started-with-github-container-registry).
+Here, we will create a protocol to discover **HTTP-based devices** that publish random sensor data.  An implementation of these devices and a discovery protocol is described in [this README in the http-extensibility branch](https://github.com/deislabs/akri/blob/http-extensibility/samples/apps/http-apps/README.md).
 
-### New DiscoveryHandler implementation
-If the resource you are interested in defining is not accessible through the [included protocols](./roadmap.md#currently-supported-protocols), then you will need to create a DiscoveryHandler for your new protocol.  For the sake of demonstration, we will create a discovery handler in order to discover mythical Nessie resources.
+Any Docker-compatible container registry will work (dockerhub, Github Container Registry, Azure Container Registry, etc).  For this sample, we are using the [GitHub Container Registry](https://github.blog/2020-09-01-introducing-github-container-registry/). You can follow the [getting started guide here to enable it for yourself](https://docs.github.com/en/free-pro-team@latest/packages/getting-started-with-github-container-registry).

```

Comment:
"will work for hosting the Akri component containers"

Reply:
"will work for hosting the components used in this example" (to include the mock device/discovery code)